observations/obs_converters/GSI2DART/radinfo.f90

```diff
@@ -34,14 +35,15 @@ subroutine radinfo_read
 !============================================================================
 
 !   Determine number of entries in satellite information file
-    open(lunin,file='satinfo',form='formatted')
+    lunin = open_file('satinfo',form='formatted',action='read')
```

Comment:
I found this to be difficult to figure out. Should we have 'satinfo' hard-coded into radinfo.f90? If, so should a copy of satinfo be included in the work directory or perhaps an explanation satinfo in the README?